### ISSUE TITLE
Seeing issue with IE11 potentially related

### ISSUE THREAD

**Issue by nspector** [inspect-js/which-typed-array#51]
Hi! On our apps on IE11 we're seeing the error: `this engine has support for Symbol.toStringTag, but Float32Array does not have the property! Please report this.` I believe it is from the changes that were made yesterday. If not, I'm sorry for the inconvenience. My theory is that the change from 
```
var hasToStringTag = hasSymbols && typeof Symbol.toStringTag === 'symbol';
```
to
```
var hasToStringTag = require('has-tostringtag/shams')();
```

is not the same check. `has-tostringtag/shams` does:
```
	return hasSymbols() && !!Symbol.toStringTag;
```
On IE11 for us, our `typeof Symbol.toStringTag ` is `object`, so it is now truthy, where-as previously it was not.

Thanks! Let me know if you want me to help with any further diagnostics!

-------- 
Edit: Confirmed that I was able to fix IE11 by pinning: `is-typed-array` and `which-typed-array` (I might only need the former, was a little whack-a-mole, and added it after just the latter didn't fix it). We're getting these as a dependency of util:
```
"util": "^0.12.4"
```

**Comment by ljharb** [comment]
Thanks for filing!

Presumably you're using `core-js`? Which version?

Can you check what `new Float32Array(0)[Symbol.toStringTag]` returns in your shimmed IE 11 environment?

**Comment by nspector** [comment]
> Presumably you're using core-js? Which version?

based on our babel config, looks like `3`.

> Can you check what new Float32Array(0)[Symbol.toStringTag] returns in your shimmed IE 11 environment?

undefined

**Comment by ljharb** [comment]
just 3? what does `node_modules/core-js/package.json` say?

**Comment by nspector** [comment]
Sorry, meant using this cofig: https://babeljs.io/docs/en/babel-plugin-transform-runtime#corejs -- the package.json shows version: `3.16.0`

**Comment by ljharb** [comment]
gotcha, thanks. i'll fix the error here to account for typed arrays with shammed symbols :-)

**Comment by nspector** [comment]
Thank you!

**Comment by ljharb** [comment]
Released in v1.1.6; please let me know if that works for you :-)

**Comment by nspector** [comment]
@ljharb coming from here now: https://github.com/inspect-js/is-typed-array/blob/main/index.js#L28

**Comment by ljharb** [comment]
aha, thanks :-) fixing that next.

**Comment by ljharb** [comment]
Fixed in `is-typed-array` v1.1.7.

**Comment by nspector** [comment]
looks like that fixed it!
